bin/problem.py

```diff
@@ -878,13 +877,8 @@ def prepare_run(problem):
         if not testcases:
             return False
 
-        if problem.interactive or problem.multi_pass:
-            validators = problem.validators(validate.OutputValidator)
-            if not validators:
-                return False
```

Comment:
If this check is removed, where do we now assert that `output_validator/` exists when the problem is interactive or multi-pass?

Reply:
We always need exactly one output validator regardless of the type and that check is still there. (the default validator takes this place if needed)